src/vivarium/framework/engine.py

```diff
@@ -297,6 +298,10 @@ def _write_results(self) -> None:
         except ConfigurationKeyError:
             self._logger.info("No results directory set; results are not written to disk.")
 
+    def write_backup(self, backup_path: Path) -> None:
+        with open(backup_path, "wb") as f:
```

Comment:
Should the backup path have a default that's handled here in vivarium?

Reply:
I suppose we could, I didnt have a great idea for what it should be